Makefile

```diff
@@ -15,7 +15,7 @@ REV=$(shell git describe --long --tags --match='v*' --always --dirty)
 # Upstream tuned daemon variables
 TUNED_REPO:=https://github.com/redhat-performance/tuned.git
 TUNED_COMMIT:=954bc4624db8fb345e9fb264ee2ac09d1736105a
-TUNED_DIR:=daemon
+TUNED_DIR:=tuned
```

Comment:
This variable is no longer needed. Let's get rid of it as we're using it only once.

Reply:
Right. Also TUNED_REPO. will remove both.

Reply:
yes, also
TUNED_COMMIT:=HEAD
Please